config/config.go

```diff
@@ -156,6 +156,11 @@ var (
 		RefreshInterval: model.Duration(5 * time.Minute),
 	}
 
+	// DefaultECSSDConfig is the default ECS SD configuration.
+	DefaultECSSDConfig = ECSSDConfig{
+		RefreshInterval: model.Duration(60 * time.Second),
```

Comment:
Regarding @beorn7 's comment: I think any discovery mechanism that uses an interval to refresh its targets is just as well off using file sd. I understand the practicality of having everything in one binary, but honestly especially in a container world where the notion of sidecar containers is well established I don't see a reason not to go that way (of course in a vm based environment).